src/dyn/dyn_variables_electronic.cpp

```diff
@@ -1290,8 +1283,7 @@ vector<double> dyn_variables::compute_average_mash_pop(int rep){
 }
 
   
-
-vector<double> dyn_variables::compute_average_sh_pop(int rep){
+vector<double> dyn_variables::compute_average_sh_pop_rep_sh1(int rep){

```

Comment:
documentation is needed - see below; also - why not combine the _sh1 and _sh0 functions in one, just passing the option 0 or 1 as the second argument?